app/src/main/java/com/f0x1d/logfox/ui/fragment/LogsFragment.kt

```diff
@@ -32,8 +32,10 @@ class LogsFragment: BaseViewModelFragment<LogsViewModel, FragmentLogsBinding>(),
 
     override val viewModel by hiltNavGraphViewModels<LogsViewModel>(R.id.logsFragment)
 
+    private val levelColorCacheMap = mutableMapOf<Int, Int>()
```

Comment:
I don't really like this to be here. Maybe it will be better to store and fill them ?lazily? in `Adapter`, or in constructor